conf/code-lists/eu-special-fiscal-territories.csv

```diff
@@ -0,0 +1,14 @@
+﻿Andorra
+Åland Islands
+Canary Islands
+French Guiana
+Guernsey
+Guadeloupe
+Jersey
+Martinique
+Mayotte
+Mount Athos
+Réunion
+Saint Martin
+San Marino
+Turkey
```

Comment:
Turkey???? Really!

Reply:
I obtained the contents of the list(s) from the ticket (and double checked with James W)